### PR TITLE
Added Proposal page and SelectedProposal Page

### PR DESCRIPTION
**Pull request by Kritika-Singhal**
Hello Mentors @devesh-verma  and @vaibhavdaren 

In this PR I have completed Desktop 5 [Proposal Page] and Desktop 6 [Selected Proposal Page]

Added files -

- ProposalDetail
- SelectedProposals
- YourProposals

Here is the Output screen -

YourProposals

YourProposals

SelectedProposals

SelectedProposal

Note: Along with the above changes, I have made OrganizationDetailPage (Desktop 4) data to fetch from JSON file. Its not hard coded anymore. This page was added in [#15 PR](https://github.com/codeuino/Design-Initiative-Dashboard-frontend/pull/15)

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -7,22 +7,23 @@ import DetailContainer from './Components/Organization/OrganizationDetail/Detail
...
 import ProjectDetails from './Components/Organization/OrganizationDetail/ProjectDetails/ProjectDetailsContainer'
-
+import SelectedProposals from './Components/Dashboard/Proposals/SelectedProposals';
+import YourProposals from './Components/Dashboard/Proposals/YourProposals';
 class App extends Component {
   render() {
     return (
       <BrowserRouter>
-	  <Navbar />
+	    <Navbar />
```
Comment (ksraj123) [inline client/src/App.js:16]: Navbar should not be outside the app, please correct indention for this as well.

File: client/src/Components/Dashboard/Proposals/ProposalDetailStyling.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { makeStyles } from "@material-ui/core/styles";
+
```
[the remark this answers is not in the document]
Comment (Kritika-Singhal) [reply client/src/Components/Dashboard/Proposals/ProposalDetailStyling.jsx:2]: I have change the extension of Styling files. Should I change extensions of other files as well which have jsx code?

File: client/src/App.js (hunk 1 above)
Comment (Kritika-Singhal) [reply client/src/App.js]: Thanks for suggesting changes. I have updated the changes. Please have a look.

File: client/src/Components/Dashboard/Proposals/ProposalDetailStyling.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ksraj123) [reply client/src/Components/Dashboard/Proposals/ProposalDetailStyling.jsx]: No, let it be.

**Pull request merged**